fabtests/scripts/runfabtests.sh

```diff
@@ -130,12 +130,10 @@ functional_tests=(
 	"fi_recv_cancel -e rdm -V"
 	"fi_unexpected_msg -e msg -I 10"
 	"fi_unexpected_msg -e rdm -I 10"
-	"fi_inj_complete -e msg"
-	"fi_inj_complete -e rdm"
-	"fi_inj_complete -e dgram"
-	"fi_inj_complete -e msg -S"
-	"fi_inj_complete -e rdm -S"
-	"fi_inj_complete -e dgram -S"
+	"fi_msg_inject -A inject -v"
+	"fi_msg_inject -N -A inject -v"
+	"fi_msg_inject -A inj_complete -v"
+	"fi_msg_inject -N -A inj_complete -v"
```

Comment:
You change runfabtests to use the new name but your rename patch comes after. Move the runfabtests rename to your rename patch